### ISSUE TITLE
False positive in Lint/FormatParameterMismatch when called on a variable

### ISSUE THREAD

**Issue by rrosenblum** [bbatsov/rubocop#2234]
Similar to #2123 and #2232

The cop will register an offense for
```ruby
CONST = '%s %s'
CONST % [1, 2]
format(CONST, 1, 2)
sprintf(CONST, 1, 2)
```

I am currently working on a fix for this.
